### DISCUSSION TITLE
Add Tide to Marine API

### DISCUSSION THREAD

**Discussion by kuwood** [open-meteo/open-meteo#1125] [Ideas]
It would be nice to also get tide information, current, and forecast etc. Really appreciate this project and all those involved!

**Comment by patrick-zippenfenig** [comment]
Hi, tide information is frequently requested. See https://github.com/open-meteo/open-meteo/issues/104. However, there has not been much progress. With recent updates large scale ocean currents have been added, but they are not suitable for small scale navigation

**Reply by kuwood** [reply]
Ah sorry for the duplication. I only searched discussions and did not think to search issues as well. Thank you for linking the issue thread and thanks for your help!

**Comment by patrick-zippenfenig** [comment]
There is some progress on ocean tides forecasts now: https://github.com/open-meteo/open-meteo/issues/104#issuecomment-2648705631
